docs/assets/js/swagger.yml

```diff
@@ -2026,10 +2043,10 @@ paths:
         - export
         - branches
       operationId: run
-      summary: export branch
```

Comment:
merge issues?

Reply:
Yup; reverted.  Thanks!

Reply:
Actually... this is just the broken half-unused copy of swagger.yml we use for JS.  @nopcoder can explain why intersecting the broken contradictory build chains of Makefile, golang and React require this.